data/macros.meson

```diff
@@ -45,3 +45,8 @@
         --num-processes %{_smp_build_ncpus} \
         --print-errorlogs \
         %{nil}}
+
+%meson_buildrequires \
+     %{shrink: python3 %{_rpmconfigdir}/mesongenbuildreq%{__meson} \
```

Comment:
Now you have the parameters messed up (missing space between mesongenbuildreq and %{__meson})

`python3 %{_rpmconfigdir}/mesongenbuildreq %{__meson} \`

ie. run python3, with the script `%{_rpmconfigdir}/mesongenbuildreq` and pass as parameter to the script the path to `%{__meson}`